draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1572,6 +1582,9 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
                  / <x:ref>DIGIT</x:ref> / <x:ref>ALPHA</x:ref>
                  ; any <x:ref>VCHAR</x:ref>, except delimiters
 </sourcecode>
+</section>
+
+<section title="Quoted Strings and Comments">
```

Comment:
I think it would be better to have separate subsections for strings and comments - having them in the same implies some relation, no?